lib/activejob/lockable/lockable.rb

```diff
@@ -27,18 +27,25 @@ def enqueue(options = {})
 
       def lock!
         return if lock_period.to_i <= 0
```

Comment:
Probably no need to call `.to_i`, since you have done it in `lock_period` method.

Reply:
The `lock_period` method can be overridden (that's in the README even). This is me being afraid of the whole untyped language thing, is it ok if I keep it?

Reply:
Yes, sure!